cangaroo.gemspec

```diff
@@ -25,6 +25,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'interactor-rails', '~> 2.0.2'
   s.add_dependency 'json-schema', '~> 2.8.0'
   s.add_dependency 'httparty', '~> 0.14.0'
+  s.add_dependency 'solidus_support', '~> 0.1.5'
```

Comment:
I think we should not add a dependency. Why should not we just do this?: 

```ruby
if Gem.loaded_specs["rails"].version >= Gem::Version.new('5.0')
  class CreateCangarooConnections < ActiveRecord::Migration[4.2]; end
else
  class CreateCangarooConnections < ActiveRecord::Migration; end
end
  		  
CreateCangarooConnections.class_eval do

```

Reply:
I'm not sure about the code you propose.

I prefer handle the version conditional inside a new module or class instead of in each migration.

We can move the code inside a module, but I think we are going to end up with something similar to [`solidus_support/migration`](https://github.com/solidusio/solidus_support/blob/master/lib/solidus_support/migration.rb)

I'm really not sure what is the problem with add a new dependency?
I feel that we are trying to reinvent the wheel.
I think the gem is not too big and they solve the migration compatibility problem very well.

Aside is less code that we should maintain.

Reply:
I agree with @bricesanchez. solidus_support gem have (and could have in the future) many other things that are not required in this gem which can also be used outside from the Solidus world.

We could create a support module here that does exactly the same as soldius_support/migration IMO. @AlessioRocco WDYT?